package.json

```diff
@@ -8,16 +8,33 @@
   "license": "MIT",
   "scripts": {
     "start": "parcel ./index.html",
-    "build": "parcel build ./index.html"
+    "build": "parcel build ./index.html",
+    "pretty": "prettier --write \"src/**/*.js\""
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{js,css}": "prettier --write",
+    "src/**/*.js": ["./node_modules/.bin/eslint", "git add"]
   },
   "dependencies": {
+    "lint-staged": "^8.0.4",
+    "prop-types": "^15.6.2",
     "react": "^16.5.2",
     "react-dom": "^16.5.2"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
```

Comment:
Why these dependencies need to be removed?

Reply:
Cool. Let's make sure we do not need them besides _inside_ parcel, in other case we will need them here again.